CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Breaking changes:
 
 Breaking changes:
 - Move enableNdk from SentryOptions to SentryAndroidOptions ([#2793](https://github.com/getsentry/sentry-java/pull/2793))
+- Fix don't overwrite the span status of unfinished spans ([#2859](https://github.com/getsentry/sentry-java/pull/2859))
```

Comment:
Also, should we mention min support for self-hosted? as per https://github.com/getsentry/sentry-java/issues/2730#issuecomment-1569843157